rust/Earthfile

```diff
@@ -13,6 +13,9 @@ INIT:
     IF [ "$CARGO_HOME" = "" ]
         ENV CARGO_HOME="$HOME/.cargo"
     END
+    IF ! echo "$PATH" | grep -q "$CARGO_HOME/bin"
```

Comment:
In my `.bashrc` I have this construct as:
```
if ! echo $PATH | /bin/egrep -q "(^|:)$1($|:)" ; then
```

Which avoids matching a path which happens to contain but is not exactly `$CARGO_HOME/bin`. Maybe doesn't matter for this specific use but though I'd would mention it.

Reply:
FWIW I think `grep -E` is the same as `egrep` (which indeed might be a Debian specific synonym)?

Reply:
yes, but my idea was that that synonym is not generally available though.

Reply:
Sure, was just saying you can probably just use `/bin/grep -Eq "(^|:)$1($|:)" `

Reply:
I see how I expressed myself badly :)
ANyway it's pushed!
Thanks!